### PR TITLE
Erdikose/fix/source code selection

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

- prevents complete source code selection on first resource selection

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/molecules/Monaco/Monaco.tsx [hunk 1]
```diff
@@ -329,11 +329,11 @@ const Monaco = (props: {editorHeight: string}) => {
   }, [selectedResourceId, resourceMap]);
 
   useEffect(() => {
-    if (editor) {
+    if (editor && code) {
```
Comment (olensmar) [inline src/components/molecules/Monaco/Monaco.tsx:332]: hm... isn't this going to set the selection every time the code changes (which is after each debounce) ? What happens if you are editing in a scrolled-down portion of the document?

Comment (erdkse) [reply src/components/molecules/Monaco/Monaco.tsx:332]: hmm, yeah! You are right I didn't think and test with the case you say.

Comment (catalinhoha) [reply src/components/molecules/Monaco/Monaco.tsx:332]: I think we need logic to detect when code changes from empty string for the first time and set the selection at the top, but not on every code change

Comment (erdkse) [reply src/components/molecules/Monaco/Monaco.tsx]: I did the similar approach

**Review by olensmar — APPROVED** [review]

**Pull request merged**